### PR TITLE
Practice Problems for Catalan Numbers

### PR DESCRIPTION
**Pull request by brebenelmihnea**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
image

friendly reminder about that convo we had

**Comment by brebenelmihnea** [comment]
:((

File: content/4_Gold/Combinatorics.problems.json [hunk 1]
```diff
@@ -45,6 +45,44 @@
       }
     }
   ],
```
Comment (ryanchou-dev) [inline content/4_Gold/Combinatorics.problems.json:47]: all starred?

File: solutions/gold/cses-2064.mdx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+	int N;
+	cin >> N;
+
+	if (N & 1) {
+		cout << 0;
+		return 0;
+	}
+
+	auto exp = [&](int x, int y) -> int {
```
Comment (ryanchou-dev) [inline solutions/gold/cses-2064.mdx:35]: std::function ? 🥺

**Review by ryanchou-dev — COMMENTED** [review]
sort of postmortem, but could you also add wikipedia as a resource so that they can view derivation/proof

**Comment by brebenelmihnea** [comment]
I'll close this pr and move all the practice problems in the new pr where i'm moving the whole section.

**Pull request closed without merging**